stream fails with "TypeError: get_child() missing 1 required positional argument: 'child_name'"

Attempting to use the 'stream' command always fails with the following stack:

> 15-12-24 19:15:11.484 [ERROR] [acd_cli] - Traceback (most recent call last):
>   File "/usr/lib64/python3.4/site-packages/acdcli-0.3.1-py3.4.egg/EGG-INFO/scripts/acd_cli.py", line  201, in wrapped
>     ret_val = f(*args, **kwargs)
>   File "/usr/lib64/python3.4/site-packages/acdcli-0.3.1-py3.4.egg/EGG-INFO/scripts/acd_cli.py", line  448, in upload_stream
>    child = cache.get_child(file_name)
> TypeError: get_child() missing 1 required positional argument: 'child_name'

(Pull request in-bound w/ one-line fix)

--- a/acd_cli.py
+++ b/acd_cli.py
@@ -445,7 +445,7 @@ def upload_stream(stream, file_name, parent_id, overwr=False, dedup=False,
     hasher = hashing.IncrementalHasher()
 
     parent = cache.get_node(parent_id)
-    child = cache.get_child(file_name)
+    child = cache.get_child(parent_id, file_name)
 
     try:
         if child and overwr:

